models/chainlink/bnb/chainlink_bnb_price_feeds.sql

```diff
@@ -1,59 +1,59 @@
-{{ config(
-    alias = alias('price_feeds'),
-    partition_by = ['block_date'],
-    materialized = 'incremental',
-    file_format = 'delta',
-    incremental_strategy = 'merge',
-    unique_key = ['blockchain', 'block_number', 'proxy_address','underlying_token_address'],
+{{
+  config(
+    tags=['dunesql'],
+    alias='price_feeds',
```

Comment:
please keep the alias macro here:
`alias = alias('price_feeds'),`

notice the non-legacy files don't pass the second param in, as it defaults to false. while the legacy files pass in the second flag and set to true.

this is needed for our orchestration to configure behind-the-scenes

Reply:
this will apply universally, only commenting here